extensions/amp-subscriptions/0.1/entitlement-store.js

```diff
@@ -83,7 +83,7 @@ export class EntitlementStore {
         }
       }
 
-      if (entitlementsResolved === this.serviceIds_.length) {
+      if (this.areAllPlatformsResolved_()) {
```

Comment:
That means this promise may never resolve if it wasn't resolved yet. You need to check `areAllplatformsResolved_` in the `onChange` below as well. That should be covered by a test as well.

Reply:
Done!